CRM/Core/ClassLoader.php

```diff
@@ -103,9 +103,11 @@ public function register($prepend = FALSE) {
       return;
     }
     $civicrm_base_path = dirname(dirname(__DIR__));
+    $composer_autoload = "$civicrm_base_path/vendor/autoload.php";
 
-    // @todo: need some way to check if we should register vendor or not
-    //require_once dirname(dirname(__DIR__)) . '/vendor/autoload.php';
+    if (file_exists($composer_autoload)) {
```

Comment:
The conditional is a good idea, although I'm wondering if we might actually need to check both locations like https://github.com/civicrm/cv/blob/v0.1.27/bin/cv#L9

Reason: for better or worse, Civi has two bootstrap processes:
 * CMS-first (typical case; ex: a page-request hits Drupal, which then boots Civi by loading `civicrm.settings.php` and `CRM_Core_Config::singleton()`)
 * Civi-first (less typical case; ex: request hits `extern/rest.php`, `extern/ipn.php`, `bin/cli.php`, or `cv`; this reads `civicrm.settings.php` and then boots CMS via `CRM_Utils_System_*::loadBootStrap()`)

In the second scenario, the classloader needs to be available.

Speculation: this should be fine as long as Civi+CMS behavior are convergent -- they both say `require_once vendor/autoload.php` for the same copy of `autoload.php`.